release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2842,8 +2842,67 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.2
+[id="ocp-4-17-2_{context}"]
+=== RHSA-2024:8229 - {product-title} {product-version}.2 image release, bug fix, and security update advisory
```

Comment:
"image release" MIGHT NOT BE NEEDED HERE. TYPICALLY STEMS FROM THE ORIGINAL RELEASE.